metadata-ingestion/tests/integration/delta_lake/golden_files/local/golden_mces_tables_with_nested_datatypes.json

```diff
@@ -1796,397 +1796,6 @@
         "lastRunId": "no-run-id-provided"
     }
 },
-{
```

Comment:
Note for the reviewers:

Removed aspects are not _lost_; they were duplicated and updating the golden file resulted on removing the duplicated aspects.